docs/DeveloperGuide.md

```diff
@@ -894,14 +894,23 @@ testers are expected to do more *exploratory* testing.
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-<div style="text-align: right"><a href="https://ay2021s1-cs2103t-f13-4.github.io/tp/DeveloperGuide.html">^ Back to top</a></div>
+<div style="text-align: right"><a href="https://ay2021s1-cs2103t-f13-4.github.io/tp/DeveloperGuide.html#">^ Back to top</a></div>
 
 ### Saving data
 
 1. Dealing with missing/corrupted data files
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+   1. Test case: Delete the `foodInventory.json` file in the `data` folder.<br>
+      Expected: Status message shows that the data file has not been found and that the application will be starting with a sample food inventory.
+      The application starts with a sample food inventory.
```

Comment:
This line and the previous sentence sounds quite repetitive.

Reply:
I stated them explicitly since one specifies the response and one the actual action. But I somewhat agree, so I am removing the last sentence.